.github/workflows/_e2e.yaml

```diff
@@ -22,7 +22,8 @@ jobs:
   e2e:
     timeout-minutes: 60
     # can't use env vars here
-    runs-on: ${{ inputs.runner }}
+    # workaround for this bug https://github.com/actions/runner-images/issues/11985 - revert when done
+    runs-on: ${{ fromJSON(inputs.matrix).k8sVersion == 'kindipv6' && (inputs.runner == 'ubuntu-latest-kong' && 'ubuntu-22.04-kong' || 'ubuntu-22.04') || inputs.runner }}
```

Comment:
I think `ubuntu-22.04-kong` doesn't exist

Reply:
Check here https://github.com/kumahq/kuma/settings/actions/runners we don't have a ubuntu-22.04 runnner